apps/src/storage/firebaseStorage.js

```diff
@@ -502,6 +504,32 @@ FirebaseStorage.copyStaticTable = function(tableName, onSuccess, onError) {
     .then(onSuccess, onError);
 };
 
+function enforceUniqueTableNames(tableName) {
+  let projectRef = getProjectDatabase();
```

Comment:
This might be my lack of knowledge on databases - should this be projectDatabase rather than projectRef? or if not, what is "ref" in this case (for my knowledge)

Reply:
`getProjectDatabase()` returns a reference to the firebase database at `v3/channels/<channel-id>`. Throughout the Firebase code it seems to be convention to name variables like this with `Ref`